sycl/doc/GetStartedGuide.md

```diff
@@ -303,6 +304,16 @@ as well as the CUDA Runtime API to be installed, see
 Currently, this has only been tried on Linux, with ROCm 4.2.0 or 4.3.0, with
 CUDA 11, and using a GeForce 1060 device.
 
+### Build DPC++ toolchain with support for ARM processors
+
+To enable support for ARM processors, follow the instructions for the linux
```

Comment:
Should we include a note here about it not being part of CI? I worry that we won't catch it if this breaks and users might be inclined to think that we officially support this target.

Tag @npmiller & @AerialMantis

Reply:
Agreed, we should have a note to that effect.

In general it should work fine as we build on top of LLVM, but we're not testing it.

Reply:
Should I add "Since this feature lacks automated testing within continuous integration, we cannot guarantee its stability in all cases." at the end of the entry, or do you have a standard comment for this situation?

Reply:
I think we can adapt the comment in the previous section to fit this section, i.e.:
> There is experimental support for oneAPI DPC++ for HIP on Nvidia devices.
There is no continuous integration for this and there are no guarantees for
supported platforms or configurations.

Reply:
Maybe we can just leave it as "There is no continuous integration for this, and there are no guarantees for supported platforms or configurations."

Reply:
Sounds good to me!